src/test/java/io/pinecone/ListEndpointValidationTest.java

```diff
@@ -0,0 +1,64 @@
+package io.pinecone;
+
+import io.pinecone.clients.Index;
+import io.pinecone.exceptions.PineconeValidationException;
+import org.junit.jupiter.api.Test;
+
+
+import static org.junit.jupiter.api.Assertions.*;
+
+public class ListEndpointValidationTest {
+    @Test
+    public void testValidateListEndpointParameters() {
+
+        PineconeValidationException thrownNullNamespace =assertThrows(PineconeValidationException.class, () -> {
+            Index.validateListEndpointParameters(null, null, null, null, true, true, true);
+        });
+        assertEquals("Namespace cannot be null or empty", thrownNullNamespace.getMessage());
```

Comment:
nit: This would be tedious but breaking things into distinct tests with names that call out what is being tested might be nice for readability.

Reply:
Done!

Reply:
Also changed so now it tests the method in the interface class (had to mock implementing class `Index`)